.stylelintrc.js

```diff
@@ -12,42 +11,24 @@ module.exports = {
     '**/docs/public/**',
     '**/docs/static/**',
   ],
-  plugins: ['stylelint-scss', 'stylelint-order'],
+  plugins: ['stylelint-order'],
   reportDescriptionlessDisables: true,
-  reportNeedlessDisables: true,
   reportInvalidScopeDisables: true,
+  reportNeedlessDisables: true,
   rules: {
-    // Turning this off in favor of 'at-rule-allowed-list'
-    'at-rule-no-unknown': null,
-    'at-rule-allowed-list': [
-      [
-        'function',
-        'if',
-        'use',
-        'media',
-        'supports',
-        'font-face',
-        'keyframes',
-        'extend',
-        'mixin',
-        'include',
-        // TODO: At some point we will be switching from 'import' to 'use'
-        'import',
-        // SCSS at-rules
-        'forward',
-        'container',
-        'each',
-      ],
-      { severity: 'warning' },
-    ],
-    // Importants should only be used in our utility classes if possible
+    'comment-empty-line-before': null,
+    'custom-property-pattern': null,
+    // `!important` should only be used in our utility classes if possible
     'declaration-no-important': [true, { severity: 'warning' }],
-    'function-no-unknown': [true, { ignoreFunctions: ['percentage', 'math.div'] }],
+    'keyframes-name-pattern': null,
```

Comment:
Linter wanted keyframe styles as kebab case names, but we don't currently have a standard on that. Can revisit later if needed.